server/elasticsearch/src/main/resources/hawkular-apm-mapping.json

```diff
@@ -17,11 +17,6 @@
             }],
             "date_detection" : false,
             "numeric_detection": false,
-            "_timestamp": {
```

Comment:
I think otherwise ES uses the time the document was recorded as a timestamp for query purposes.